src/character.cpp

```diff
@@ -913,7 +913,7 @@ aim_mods_cache Character::gen_aim_mods_cache( const item &gun )const
 
 double Character::fastest_aiming_method_speed( const item &gun, double recoil,
         const Target_attributes target_attributes,
-        std::optional<std::reference_wrapper<const parallax_cache>> parallax_cache ) const
+        const std::optional<std::reference_wrapper<parallax_cache>> parallax_cache ) const
```

Comment:
This can simply be a pointer which is by definition an "optional reference"...

Reply:
I knew, but I want to make the code's meaning more explicit to those who reads the code so for now I often avoid using nullptr to indicate "no cache"...

Reply:
An hour has passed and there is no further debate, so leave it as is. Close this discussion.